play-media-native/features/ContentItemFields/ContentItemFields.tsx

```diff
@@ -248,6 +251,7 @@ export const ContentItemFields = ({
 
           return renderer(overrideKey);
         })}
+        {showRequiredBanner && <RequiredFieldsBanner />}
```

Comment:
Required fields message should be at the top on Edit event details, Edit athlete details screens.

Currently it is at the bottom
![image](https://user-images.githubusercontent.com/96472119/226675998-11f6ec2b-57e7-4cb5-bfdc-9acff97b27a5.png)